tracer.py

```diff
@@ -197,8 +194,8 @@ def postprocess_rays(init_points, final_points, cells, inf_distance=1000):
     print("postprocessing ray list")
     for i, point in tqdm.tqdm(enumerate(final_points), unit ="rays", position=0, unit_scale=True):
         if not point.all():
-            if (final_points[i-1] == init_points[i]).all():  # tracable ray
-                direction = final_points[i-1] - init_points[i-1]
+            direction = final_points[i-1] - init_points[i-1]
+            if not direction.all() == 0:  # tracable ray
```

Comment:
This ensures that no degenerate rays are processed. Not sure why you are seeing the problem.